src/deepagents/graph.py

```diff
@@ -67,11 +80,20 @@ def create_deep_agent(
         state_schema
     )
     all_tools = built_in_tools + list(tools) + [task_tool]
+    
+    # Should never be the case that both are specified
+    selected_post_model_hook = None
+    if post_model_hook is not None:
+        selected_post_model_hook = post_model_hook
+    elif interrupt_config:
```

Comment:
nit: good practice to try to end with a `else` so reader can understand fully
lets maybe put the raising of the error here so logic is all together (easier to read/understand)